lib/Tga/SimHash/Tokenizer/String128Tokenizer.php

```diff
@@ -18,13 +18,20 @@
  */
 class String128Tokenizer implements TokenizerInterface
 {
+ 
+    protected static $search = array('0','1','2','3','4','5','6','7','8','9','a','b','c','d','e','f');
```

Comment:
You shouldn't add protected variables like this without a specific need as they will need to be handled by BC promise.

Reply:
fix